hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -5,14 +5,243 @@ title: "Life Cycle Controller Technical Specification"
+
+This signal thus needs to be routed to all security-aware and debug capable peripherals.
+This signal is used to determine whether OpenTitan peripheral register interfaces should [silently error]({{< relref "doc/rm/register_tool/_index.md#error_responses" >}}).
+If HW_DEBUG_EN is set to ON, normal errors should be returned.
+If HW_DEBUG_EN is set to OFF, errors should return silently.
+
+Similar to DFT_EN, HW_DEBUG_EN is also used to isolate the processor TAP.
+When HW_DEBUG_EN is OFF, the TAP should not be able to perform its' normal debug access, thus preventing an external entity from hijacking the processor.
+
+### CPU_EN
+
+CPU enable controls whether code execution is allowed.
+This is implemented as part of the processor's reset controls.
+In OpenTitan's [reset topology]({{< relref "hw/ip/rstmgr/doc" >}}), it is not possible to reset only the processor by itself, so this reset control extends to a large population of the OpenTitan peripherals.
+
+This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it is not possible for the processor to execute code that breaks the device out of a non-functional state.
+
+In conjunction with DFT_EN / HW_DEBUG_EN, this acts as the final layer in life cycle defense in depth.
+
+### PROVISION_EN
+
+Provision-enable is a separate signal that controls whether the non-volatile provisioning of life cycle related collateral can be accessed (see [OTP]({{< relref "#otp-collateral" >}}) and [flash]({{< relref "#flash-collateral" >}}) sections for list of collateral and also what collateral in each area is affected by provision-enable).
+This satisfies the dependency requirement between the manufacturing state and the identity state.
+
+Provision-enable is active only during DEV / PROD / PROD_END / RMA.
+During other states, it is not possible to either read or modify the collateral.
+This specifically limits the danger of rogue software images during any TEST_UNLOCKED state.
+
+However, as provision-enable only gates functional access and not DFT access, it is still possible for a malicious agent to bypass this protection by abusing scan shift/capture mechanics.
+
+Note also, provision-enable is a blanket control over all provision related collateral.  Each collateral contains more specific OTP / Flash / Software based decoding that further limits accessibility.
+See the Flash and OTP sections for more details.
+
+### KEY_MANAGER_EN
+
+The Key manager enable signal allows the key manager to function normally.
+When this signal is logically disabled, any existing key manager collateral is uninstantiated and wiped; further instantiation and generation calls for the key manager are also made unavailable.
+
+Key manager enable is active only during DEV / PROD / PROD_END / RMA.
+
+### CLK_BYP_EN
+
+The clock bypass signal switches the main system clock to an external clock signal in RAW and TEST_LOCKED states when initiating a life cycle transition
```

Comment:
I think you might need a period at the end of this sequence
The clock bypass signal -> The clock bypass enable signal

Reply:
fixed, thanks.